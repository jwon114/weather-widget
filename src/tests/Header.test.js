import React from 'react'
import renderer from 'react-test-renderer';
import Header from '../components/Header';

test('renders Header', () => {
  const tree = renderer
    .create(<Header 
              location={'London'} 
              datetime={'1591092083'}
              summary={'cloudy with a chance of meatballs'}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});