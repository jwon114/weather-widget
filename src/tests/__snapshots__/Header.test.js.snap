// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header 1`] = `
<div
  className="header__container"
>
  <div
    className="header__location"
  >
    London
  </div>
  <div
    className="header__time"
  >
    Tuesday
     
    11:01 AM
  </div>
  <div
    className="header__summary"
  >
    cloudy with a chance of meatballs
  </div>
</div>
`;
